src/scrabble.js

```diff
@@ -1,5 +1,48 @@
-function scrabble() {
-  // write code here
+function scrabble(word) {
+  const letterValue = {
+    A: 1,
+    E: 1,
+    I: 1,
+    O: 1,
+    U: 1,
+    L: 1,
+    N: 1,
+    R: 1,
+    S: 1,
+    T: 1,
+    D: 2,
+    G: 2,
+    B: 3,
+    C: 3,
+    M: 3,
+    P: 3,
+    F: 4,
+    H: 4,
+    V: 4,
+    W: 4,
+    Y: 4,
+    K: 5,
+    J: 8,
+    X: 8,
+    Q: 10,
+    Z: 10
+  }
+  console.log(letterValue)
+  if (!word || typeof word !== 'string') {
+    return 0
+  } else {
+    let score = 0
+    const letter = word.toUpperCase().split('')
+    for (let i = 0; i < letter.length; i++) {
```

Comment:
Since `letter` is an array, you could have used an array method here instead such as `.forEach()`

Reply:
Yes you are right , but i used object in the start so array method or property made a problem here so for get rid of that i used this.